app/dashboard/templatetags/site.py

```diff
@@ -0,0 +1,13 @@
+from django import template
+#TODO: CHANGE to:
+#from django.contrib.sites.models import Site
+from django.conf import settings
+
+register = template.Library()
+
+
+@register.simple_tag
+def current_domain():
+    #TODO: CHANGE to:
+    #return 'http://%s' % Site.objects.get_current().domain
```

Comment:
still TODO?

Reply:
@owocki I wanted to make sure the optional ```Sites``` framework is being used on the production gitcoin server. See here:
https://docs.djangoproject.com/en/2.1/ref/contrib/sites/
Otherewise, this TODO should be erased, and the domain will have to come from the settings file.